<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>1221</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="lay_table">
    <item>
     <widget class="QTableView" name="tbl_info"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1221</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="lay_items">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>날짜 </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="date_select_from">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="date_select_to">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>아이템ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_item_id">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>아이템명</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_item_name">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>생산오더</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_prod_id">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>판매오더</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_sales_id">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>오더상태</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comb_prod_status">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>릴리스</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>시작됨</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_search">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>조회</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
